.travis.yml

```diff
@@ -3,18 +3,10 @@ cache: pip
 dist: trusty
 python:
   - "3.6"
-addons:
-  apt:
-    sources:
-      - boost-latest
-      - ubuntu-toolchain-r-test
-    packages:
-      - libboost-system1.55-dev
-      - libboost-filesystem1.55-dev
-      - g++-4.8
+  - "3.7"
 
 before_install:
-  - pip install pip --upgrade
+  - pip install pip pytest --upgrade
 
 install:
   # annoyingly we can't put this in setup.py...
```

Comment:
We should now be able to use `pint` from `pypy` as they have a new release: https://github.com/hgrecco/pint/releases

Reply:
(failing of the current travis build seem to be related to new commits on `pint`'s master...)